hoomd/hpmc/compute.py

```diff
@@ -161,15 +161,25 @@ class SDF(Compute):
 
     For each pair of particles :math:`i` and :math:`j` `SDF` scales the particle
     separation vector by the factor :math:`(1-x)` and finds the smallest
-    positive value of :math:`x` leading to an overlap of the particle shapes:
+    positive value of :math:`x` leading to either an overlap of the particle
+    shapes (a "hard overlap") or a change in the pair energy
+    :math:`U_{\mathrm{pair},ij}` (a "soft overlap"):
 
     .. math::
 
         x_{ij}(\vec{A}) = \min \{ x \in \mathbb{R}_{> 0} :
             \mathrm{overlap}\left(
                 S_i(\mathbf{q}_i),
-                S_j(\mathbf{q}_j, (1-x)(\vec{r}^t_j - (\vec{r}_i + \vec{A})))
-            \right) \ne \emptyset \}
+                S_j(\mathbf{q}_j, (1-x)(\vec{r}_j - (\vec{r}_i + \vec{A})))
+            \right) \ne \emptyset
+            \\
+            \lor U_{\mathrm{pair},ij}((1-x)(\vec{r}_j - (\vec{r}_i + \vec{A})),
+                                 \mathbf{q}_i,
+                                 \mathbf{q}_j)
+                \ne
+            U_{\mathrm{pair},ij}(\vec{r}_j - (\vec{r}_i + \vec{A}),
+                                 \mathbf{q}_i,
+                                 \mathbf{q}_j) \}
```

Comment:
This run-on equation is hard to read. I suggest spacing it onto more lines and using alignment so the terms of the "or" operation line up. See for example, the U_pair and U_shape equations at https://hoomd-blue.readthedocs.io/en/v3.3.0/module-hpmc-integrate.html

Reply:
I broke it into several lines now. It's definitely easier to read, but perhaps takes up more vertical space than needed. Feel free to change.